gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -1078,23 +1078,6 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("to_unix_timestamp")
   enableSuite[GlutenDeprecatedAPISuite]
   enableSuite[GlutenDynamicPartitionPruningV1SuiteAEOff]
-    .exclude("SPARK-38674: Remove useless deduplicate in SubqueryBroadcastExec")
-    .excludeByPrefix("static scan metrics")
-    .excludeByPrefix("Gluten - static scan metrics")
-  enableSuite[GlutenDynamicPartitionPruningV1SuiteAEOn]
```

Comment:
these lines of enabling suite should not be removed

Reply:
done